plugins/ibm_qradar/icon_ibm_qradar/util/utils.py

```diff
@@ -0,0 +1,138 @@
+import json
+import logging
+
+from insightconnect_plugin_runtime.exceptions import (
+    ConnectionTestException,
+    PluginException,
+    ClientException,
+)
+from requests import Response
+
+from icon_ibm_qradar.util.constants.constant import (
+    SUCCESS_RESPONSE_CODE,
+)
+from icon_ibm_qradar.util.constants.messages import INVALID_RANGE_FOUND
+from icon_ibm_qradar.util.url import URL
+
+
+def prepare_request_params(
+    params: {},
+    logger: logging.Logger,
+    url_obj: URL,
+    input_fields: list,
+    query_params: dict = None,
+):
+    """Parse the common input provided including range, sort, filter and fields.
+
+    :param params: Input Dict
+    :param logger: Logging logger
+    :param url_obj: URL object
+    :param input_fields: Possible input fields for the action
+    :param query_params: pre set query params
+    :return:
+    """
+    if not query_params:
+        query_params = {}
+
+    headers = get_default_header()
+    basic_url = url_obj.get_basic_url()
+
+    for field in input_fields:
+        query_value = params.get(field, "")
+
+        if field == "range" and query_value != "":
+            check, query_range = validate_query_range(query_value)
+            if not check:
+                logger.error(query_range)
+                logger.info("Terminating: query_range provided is invalid")
```

Comment:
```suggestion
                logger.info("Terminating: Query Range provided is invalid")
```
 If query_range is an input, it should use the user-facing title in the log message